setup.py

```diff
@@ -39,6 +39,14 @@ def get_output(*args, **kwargs):
 # get the compile and link args
 kc = "krb5-config"
 posix = os.name != 'nt'
+
+# According to the Python documentation, in order to get the '64-bitness' of
+# the current interpreter, it is more reliable to query the sys.maxsize
+# attribute:
+#
+# Reference:
+# https://docs.python.org/3/library/platform.html#platform.architecture
+is64bits = sys.maxsize > 2**32
```

Comment:
(Same here.)

Reply:
On closer look at the code, we're using `sys.maxsize > 2**32` for testing 64-bit code execution in the setup script.
Hence - using this variable at all those places now.